models/WorkspaceUser.py

```diff
@@ -0,0 +1,28 @@
+from google.cloud import ndb
+from models import get_db
+
+
+client = get_db()
+
+
+class WorkspaceUser(ndb.Model):
+    id_workspace = ndb.IntegerProperty()
+    id_user = ndb.IntegerProperty()
+
+    # class methods (ordered by alphabet)
+    @classmethod
+    def create(cls, id_workspace, id_user):
+        with client.context():
+            # check if there's any entries with the same data already
+            workspace_id_check = cls.query(cls.id_workspace == id_workspace).get()
+            user_id_check = cls.query(cls.id_user == id_user).get()
+
+            if not workspace_id_check and user_id_check:  # if entry does not yet exist, create one
+
+                # create the workspace object and store it into Datastore
+                workspace_user = cls(id_workspace=id_workspace, id_user=id_user)
+                workspace_user.put()
+
+                return "Success"  # succes, workspace, message
```

Comment:
tole mora vrniti tri stvari:

1. Boolean, ki pove, ali je bil query uspešen (v tej vrstici True, pod else pa False)
2. WorkSpace user objekt (workspace_user, pod else pa None)
3. Sporočilo (string): "Success" v tej vrstici, po else pa tisto sporočilo, ki je že tam ("Workspace and user relation...")

Reply:
Izgledalo bo tako:

```python
return True, workspace_user, "Success"
```